<ui version="4.0" >
 <class>CalibrateFurnaceDialog</class>
 <widget class="QDialog" name="CalibrateFurnaceDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Наастройка режима работы термокамеры</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Уставка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbSetPoint" >
       <property name="suffix" >
        <string>°C</string>
       </property>
       <property name="maximum" >
        <number>1000</number>
       </property>
       <property name="singleStep" >
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbStart" >
       <property name="text" >
        <string>Старт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStop" >
       <property name="text" >
        <string>Стоп</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbExit" >
       <property name="text" >
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QwtPlot" name="qwtpData" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&lt;b>Пост. интегр.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTIntegr" >
       <property name="text" >
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>&lt;b>Пост. дифф.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTDiff" >
       <property name="text" >
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>&lt;b>Зона проп.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lPb" >
       <property name="text" >
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbSave" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>CalibrateFurnaceDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>401</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel" >
     <x>247</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
